lib/classification_lifecycle.rb

```diff
@@ -118,6 +119,10 @@ def add_project_live_state
     update_classification_metadata(:live_project, project.live)
   end
 
+  def add_user_groups
+    update_classification_metadata(:user_group_ids, user.non_identity_user_group_ids)
```

Comment:
Assignments aren't coming until later this year. But because Amy is (going to be) working on a PFE interface for creating / joining groups, I imagine we'll have groups of friends becoming groups, so having the groups for all projects might give neat insights with respect to clusters of users? Doing it in the backend also means we get it for custom frontends for free.